### PR TITLE
Ncgames 6

### PR DESCRIPTION
**Pull request by malik9153**

### CODE REVIEW

**Review by Harrison365 — CHANGES_REQUESTED** [review]
Hey Maliek :)
Please...
-Make sure that comments are on a key of comments, not comment (i know its picky).
-Use a forEach to test the keys of each comment, rather than typing each. 
-You will also need to test the length of the return array to avoid false positives.
-You will also need to test for a review that exists but has no comments.
I like that you have created a separate db.query to check ID exists :) 

Let us know when you need us to take another look. Cheers.

File: __model/Reviews.model.js
Comment (hevdevs) [inline __model/Reviews.model.js]: remove console log before merge

File: __tests__/app.test.js
Comment (hevdevs) [inline __tests__/app.test.js]: make sure you're returning db.end() in your afterAll func

**Review by hevdevs — APPROVED** [review]
Good to merge after making these little changes! Also, can you rename your model functions to be camelCase, PascalCase is more often used for classes in javascript so naming your models similar to SelectCategories instead of selectCategories is a bit misleading

**Pull request merged**
